packages/snaps-utils/src/address.ts

```diff
@@ -0,0 +1,19 @@
+import type { CaipAccountId, CaipChainId } from '@metamask/utils';
+import { parseCaipChainId, toCaipAccountId } from '@metamask/utils';
+
+/**
+ * Create a list of CAIP account IDs from an address and a list of scopes.
+ *
+ * @param address - The address to create the account IDs from.
+ * @param scopes - The scopes to create the account IDs from.
+ * @returns The list of CAIP account IDs.
+ */
+export function createAddressList(
```

Comment:
Seems like this returns account IDs, rather than addresses?
```suggestion
export function createAccountList(
```